doc/source/evaluate.rst

```diff
@@ -61,7 +61,7 @@ Submission format
 ^^^^^^^^^^^^^^^^^^^^^^
 
 To evaluate your algorithms on the BDD100K detection benchmark, you may prepare predictions in bitmask format,
-which is illustrated in `Label Format <https://doc.bdd100k.com/format.html#segmentation-label-formats>`_.
```

Comment:
It is better to use relative links for references within the document so that it can be self-contained. An example: https://github.com/bdd100k/bdd100k/blame/master/doc/source/download.rst#L6